core/domain/stats_jobs_continuous.py

```diff
@@ -64,16 +67,21 @@ class StatisticsAggregator(jobs.BaseContinuousComputationManager):
     """
     @classmethod
     def get_event_types_listened_to(cls):
+        """ Returns a events type of start exploration and complete exploration
+        """
         return [
             feconf.EVENT_TYPE_START_EXPLORATION,
             feconf.EVENT_TYPE_COMPLETE_EXPLORATION]
 
     @classmethod
     def _get_realtime_datastore_class(cls):
+      """  It will give realtime from datastore """
```

Comment:
Don't leave space at the beginning after """. Write 'Returns:' and add period(.) at the end. You can even refer to previous PR for guidance like [this](https://github.com/oppia/oppia/pull/2438/files).